mee/times.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright 2018 SiFive, Inc */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#ifndef MEE__TIMES_H
+#define MEE__TIMES_H
```

Comment:
I think these should just be in `mee/timer.h`, as they're really not their own interface.